### DISCUSSION TITLE
DeepStack DenoisingAutoEncoder

### DISCUSSION THREAD

**Discussion by sorenmacbeth** [manujosephv/pytorch_tabular#187] [Q&A]
Hello,

I'm interesting in implementing a DeepStack DAE model, which is the default model in https://github.com/ryancheunggit/tabular_dae for example. Is this possible with the current DenoisingAutoEncoderModel class and I am just missing something (it looks to be a bottleneck model to my understanding)?

**Comment by manujosephv** [comment]
@sorenmacbeth The Self supervised pipeline is very new and not mature. there is a lot of improvements to me made in there.. 

And, yes. right now the structure is a bottle neck model and is borrowing a lot of code from the repo you mentioned.

I guess you can make it a Deepstack model by just making the decoder identity. I haven't tried it out. If that is not working, feel free to make a contribution.
